com.unity.renderstreaming/Documentation~/en/faq.md

```diff
@@ -49,15 +51,15 @@ Currently, version 1.0 does not support simulcast.
 
 ### Can you broadcast the feed from multiple cameras?
 
-Currently, version 1.0 cannot.
+It is available. If you use the hardware encoder, make sure the limitation of the number of cameras using at the same time. See the NVIDIA [document]](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).
```

Comment:
Yes. If you use the hardware encoder, make sure to check the limitation of the number of cameras used at the same time. See the NVIDIA [document]](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).